client/src/components/InputSelect/InputSelect.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import "./InputSelect.css";
+import PropTypes from "prop-types";
+
+// Class Based React Component
+class InputSelect extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      classList: "InputSelect",
```

Comment:
If you just have this one class, then you can put the string in your JSX for the `className` property